pkg/scheduler/apis/config/scheme/scheme.go

```diff
@@ -33,14 +33,15 @@ var (
 )
 
 func init() {
-	AddToScheme(Scheme)
+	utilruntime.Must(AddToScheme(Scheme))
 }
 
 // AddToScheme builds the kubescheduler scheme using all known versions of the kubescheduler api.
-func AddToScheme(scheme *runtime.Scheme) {
+func AddToScheme(scheme *runtime.Scheme) error {
 	utilruntime.Must(config.AddToScheme(scheme))
 	utilruntime.Must(configv1.AddToScheme(scheme))
 	utilruntime.Must(scheme.SetVersionPriority(
 		configv1.SchemeGroupVersion,
 	))
```

Comment:
in that case, remove utilruntime.Must from these and return the errors.

Reply:
+1, return error explicitly in this case.

Reply:
em... that would be a little ugly, a lot of error handlings, which is contrary to my original intention. I found I can import the Scheme directly for reuse, no need to call the `AddToScheme` function, then I'll close this. Sorry for your time.
/close